aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,10 +1,177 @@
+
+The following image shows the same timing view of the *About* view shown previously, but this time with bundling and minification enabled.
+
+.. image:: bundling-and-minification/_static/03-MinTiming.png
+
+Minification
+------------
+Minification performs a variety of different code optimizations to reduce the size of requested assets (such as CSS, image, JavaScript files). Common results of minification include removing unnecessary white space and comments, and shortening variable names to one character. 
+
+Consider the following JavaScript function:
+
+.. code-block:: javascript
+
+	AddAltToImg = function (imageTagAndImageID, imageContext) {
+		///<signature>
+		///<summary> Adds an alt tab to the image
+		// </summary>
+		//<param name="imgElement" type="String">The image selector.</param>
+		//<param name="ContextForImage" type="String">The image context.</param>
+		///</signature>
+		var imageElement = $(imageTagAndImageID, imageContext);
+		imageElement.attr('alt', imageElement.attr('id').replace(/ID/, ''));
+	}
+
+After minification, the function is reduced to the following:
+
+.. code-block:: javascript
+
+	AddAltToImg=function(t,a){var r=$(t,a);r.attr("alt",r.attr("id").replace(/ID/,""))};
+	
+In addition to removing the comments and unnecessary whitespace, the following parameters and variable names were renamed (shortened) as follows:
+
+==================  =======  
+Original            Renamed      
+==================  =======  
+imageTagAndImageID  t  
+imageContext        a  
+imageElement        r   
+==================  =======  
+
+Impact of Bundling and Minification
+-----------------------------------
+
+The following table shows several important differences between listing all the assets individually and using bundling and minification based on the above *About* page.
+
+==================  ==========  ============  ============  
+Action              With B/M    Without B/M   Change    
+==================  ==========  ============  ============  
+File Requests       7           18            157%
+KB Transferred      156         264.68        70%
+Load Time (MS)      885         2360          167%  
+==================  ==========  ============  ============  
+
+The bytes sent had a significant reduction with bundling as browsers are fairly verbose with the HTTP headers that they apply on requests. The load time shows a big improvement, however this example was run locally. You will get greater gains in performance when using bundling and minification with assets transferred over a network. To simulate the transfer time of a slow network, you can use the `Fiddler <http://www.telerik.com/fiddler>`_ tool. Within Fiddler, select the **Fiddler Rules** menu, **Performance** > **Simulate Modem Speeds**.
+
+Controlling Bundling and Minification
```

Comment:
I think you need to insert a section in here that describes how bundling and minification is done for ASP.NET Core apps. There isn't any runtime support for bundling and minification in ASP.NET Core. Instead you handle bundling and minification at build time using third party tools like gulp/grunt with their related tasks (ex concat, uglify). You then leverage the environments support in ASP.NET Core (dev vs prod) to control when minified docs are used. You should be able to heavily leverage the bundling and minification content that is already in the [gulp](http://docs.asp.net/en/latest/client-side/using-gulp.html) doc.